requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Core deps.
-tensorflow==2.11
-tensorflow-text==2.11
+tensorflow~=2.11.0
```

Comment:
What's the difference between `~=2.11.0` and `==2.11`?

Reply:
`==2.11` is equivalent to saying `2.11.0`.  When we say `~=2.11.0`, it will update to the nearest patch release of `2.11.1`, if that is the last patch release